### DISCUSSION TITLE
Module Todo: Open / Misconfigured Fileshares

### DISCUSSION THREAD

**Discussion by TheTechromancer** [blacklanternsecurity/bbot#725] [Ideas]
NFS / SMB / FTP / iSCSI

SMB:

- [ ] Anonymous logon
- [x] SMB signing
- [x] SMB versioning

**Comment by aconite33** [comment]
SMB signing and SMB Versioning added in commit: https://github.com/blacklanternsecurity/bbot-private/commit/13e18824463059de99d9b0c27fa1b9533bc9a9a6

**Comment by SpamFaux** [comment]
Recommend handing off to a jr dev to get the module built. Assigning to @TMDeal and @Mercury0
